File: Python/Exercicios/Ex14.py
# Entrar com o peso e a altura de uma determinada pessoa. Após a digitação, exibir se esta pessoa está ou não com seu peso ideal. Fórmula: peso/altura².

p = float(input('Digite o seu peso: '))
a = float(input('Digite a sua altura no formato x.xx: '))

imc = p/(a*a)

if (imc < 20):
	print(f'Você está abaixo do peso, pois seu imc deu {imc:.2f}')

elif (imc > 20) and (imc < 25): 
	print(f'Você está com peso ideal, pois seu imc deu {imc:.2f}')

else :
	print(f'Você está acima do peso, pois seu imc deu {imc:.2f}')

File: JAVA/exjava1111/src/exercicios/Ex14.java
package exercicios;

import java.util.Scanner;

public class Ex14 {
	public static void main(String[] args) {
		//  Entrar com o peso e a altura de uma determinada pessoa. Após a digitação, exibir se esta pessoa está ou não com seu peso ideal. Fórmula: peso/altura².
	
	Scanner ler = new Scanner (System.in);
	
	int p, h, imc;
	
	System.out.printf("Digite o valor do peso: ");
	p = ler.nextInt();
	
	System.out.printf("Digite o valor da altura: ");
	h = ler.nextInt();
	
	imc = p/(h*h);
	
	
	System.out.printf("O valor do imc é %d", imc);
	
	
	

	}
}